app/src/androidTest/java/au/com/shiftyjelly/pocketcasts/models/db/FolderDaoTest.kt

```diff
@@ -0,0 +1,142 @@
+package au.com.shiftyjelly.pocketcasts.models.db
+
+import android.graphics.Color
+import androidx.room.Room
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import au.com.shiftyjelly.pocketcasts.models.db.dao.FolderDao
+import au.com.shiftyjelly.pocketcasts.models.type.PodcastsSortType
+import au.com.shiftyjelly.pocketcasts.utils.FakeFileGenerator.fakeFolder
+import junit.framework.TestCase.assertEquals
+import junit.framework.TestCase.assertNotNull
+import junit.framework.TestCase.assertNull
+import junit.framework.TestCase.assertTrue
+import kotlinx.coroutines.test.runTest
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class FolderDaoTest {
+    private lateinit var folderDao: FolderDao
+    private lateinit var testDatabase: AppDatabase
+
+    @Before
+    fun setupDatabase() {
+        val context = InstrumentationRegistry.getInstrumentation().targetContext
+        testDatabase = Room.inMemoryDatabaseBuilder(context, AppDatabase::class.java).build()
+        folderDao = testDatabase.folderDao()
+    }
+
+    @After
+    fun closeDatabase() {
+        testDatabase.clearAllTables()
+        testDatabase.close()
+    }
+    @Test
```

Comment:
Nitpick: Can you reformat the file so that newlines are added after each test? Not sure why it wasn't caught by Spotless 🤔

Reply:
done